package-system/OpenXR/build_config.json

```diff
@@ -60,7 +60,8 @@
             ],
             "custom_test_cmd" : [
            ]
-         }
+         },
+	 "Linux-aarch64": "@Linux"
```

Comment:
will this just use the x64 binaries for the aarch64? Will that work?

Reply:
Its an alias for situations where the build settings for the platform are identical. For this library, the aarch64 detection, etc is part of the native project's build scripts.

Reply:
Here is an example if there is a difference ( https://github.com/o3de/3p-package-source/pull/135  ) 
This case the dependent binary is an aarch64 one specifically.

Reply:
>For this library, the aarch64 detection, etc is part of the native project's build scripts.

Nice! Didn't know the library detects it an uses the right build parameters automatically. Thanks. I'll approve.